Adding sample configurations for various DSC resources (#1898)
microsoft/devhome: sampleConfigurations/DscResources/README.md

## Sample Configurations for Specific DSC Resources

> The sample configurations provided in this directory showcase how to create a configuration yaml that utilize DSC resources for more specific scenarios.

Reviewer: ```suggestion
The sample configurations provided in this directory showcase how to create WinGet configuration files utilizing DSC resources for more specific scenarios.

```

### [GitDsc](https://www.powershellgallery.com/packages/GitDsc/0.1.2-alpha)

Supports cloning a new repository and adding/removing remote connections to other repositories.

### [Microsoft.WindowsSandbox.DSC](https://www.powershellgallery.com/packages/Microsoft.WindowsSandbox.DSC/0.1.1-alpha)

> Create a new instance of Windows Sandbox by either providing a custom .wsb file or specifying parameters.

Reviewer: ```suggestion
Create a new instance of Windows Sandbox by either providing a custom .wsb file or specifying parameters.
>Note: [Windows Sandbox](https://learn.microsoft.com/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview#prerequisites) requires Windows 10 Pro or Enterprise, build version 18305 or Windows 11.
```
